sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/mongodb.yaml

```diff
@@ -31,7 +31,7 @@ provider:
           # - $(password)
           # - #{password}#
           patternNot:
-            "(?i)\
+            - "(?i)\
             ^(\\$[a-z_]*)?(db|my)?_?pass(word|wd)?|\
```

Comment:
Only this one requires the (?i) there, the rest of patterns look case insensitive - except the "%s" one

When fixing this, don't bother with adding the comments, I can take care of it in my next review